Content.Shared/ChangeNameInContainer/ChangeNameInContainerComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Shared.Whitelist;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.ChangeNameInContainer;
+
+[RegisterComponent, NetworkedComponent, Access(typeof(ChangeNameInContainerSystem))]
```

Comment:
document please

Reply:
I always forget to add comments to my components 😭... Fixed!